google/ggrc-core
# GGRC-7192 Send comment notifications for program

# Dependencies

This PR is `on hold` until the dependencies are merged:

- [ ] GGRC-7203 - FE part

# Issue description

Notifications on comment creation should be send in daily digest for programs.

# Steps to test the changes

1. Create a Program
2. Add a comment with checked box about 'notify contacts'
3. Check that daily digest contains comment notifications for listed roles.

1. Choose existing program before migrations
2. Add a comment with checked box about 'notify contacts'
3. Check that daily digest contains comment notifications for listed roles.

# Solution description

Make program `Commentable`, implement tests

# Sanity checklist




# PR Review checklist
